### ISSUE TITLE
[DevTools Bug]: Components Tab does not show up

### ISSUE THREAD

**Issue by antonisprovidakis** [facebook/react#25806]
### Website or app

https://beta.reactjs.org/

### Repro steps

1. Visit website
2. Open dev tools

This happens on https://beta.reactjs.org/ but I first noticed in on a personal project (localhost).

When I open the dev tools, the CPU goes up. At first, the Components tab does not show up. After a loooooong time, it does show up, however when I click on it nothing renders inside.

![Screenshot 2022-12-05 at 12 42 04](https://user-images.githubusercontent.com/58694408/205619186-85451e15-9b2f-43d9-b7f6-3ffaba6e7346.png)

I don't know if it's the newest Chrome version or the newest extension version that's causing it.

### How often does this bug happen?

Every time

**Comment by antonisprovidakis** [comment]
On my personal project, after a long time waiting the Components tab did show up. I clicked on it, and after a while this renders:

![Screenshot 2022-12-05 at 12 49 36](https://user-images.githubusercontent.com/58694408/205619385-2316a0df-b036-46d5-b487-5d364d560022.png)

**Comment by antonisprovidakis** [comment]
Some times this show up too:
```
```

**Comment by antonisprovidakis** [comment]
One more update: I installed the React DevTools extension v4.26.1 as an unpacked, and it still does not work

**Comment by mondaychen** [comment]
I'm not able to reproduce this issue. Can you let me know which Chrome and DevTools version you are using?
Also, you can try restart Chrome, or try disable other Chrome extensions and see if the issue still exists.

**Comment by antonisprovidakis** [comment]
I found a workaround to "force" the dev tools to load. Please see below:

https://user-images.githubusercontent.com/58694408/205744106-057425e8-7859-4a94-82f1-d4b345b03488.mp4

**Comment by mondaychen** [comment]
@antonisprovidakis That's interesting... It seems to be related to how Chrome launch service worker. I'll look into this and see if I can find a way to reproduce it.

**Comment by SmoothJM** [comment]
> 

I cannot click service work link. It does not show a new window : ( 
and that shows like: service work (invalid)

**Comment by afterwind-io** [comment]
Still not functional with:

Chrome: Version 108.0.5359.124 (Official Build) (x86_64)
DevTools: 4.27.1 (12/6/2022)

And for anyone who encountered the same problem, I found a somewhat "stable" workaround:
- (Optional) Fix the react devtool icon to the toolbar;
- Open the page;
- Open the devtool;
- If react devtool tab does not show up, click the extension icon, then the devtool tab should show up now;
- Click the "Components" tab;
- If the tab says "loading blah blah", close the inspector first and open it again. The devtool tab should be blank now;
- If the tab is blank, click the extension icon multiple times until something shows up, and the devtool should be seemingly working now, but with high CPU consumption.

Now it's the important part:
- Open `chrome://inspect/#service-workers` in a new browser tab;
- If all is "working" now, you should see two react devtool service worker running here(`chrome-extension://fmkadmapgofadopljbjfkapdkoienihi/build/background.js
`);
- *Swiftly* terminate BOTH one;
- Back to the page you just opened. Leave the inspector open, and the react devtool tab open and active(Important!);
- Refresh the page;
- If all is done right, the react devtool should work normally. You can open Chrome's task manager and check whether there is only one service worker running at the moment (And that is actually what above operations are all about).

So what I suspect is that due to unknown reason, the react devtool service worker instanciates twice, which causes all those problems. I hope this can help out finding the issue, cause the problem is REALLY annoying.

**Comment by tassoevan** [comment]
@afterwind-io I followed the instructions, but nothing happened. Then, instead of terminating those pesky service workers, I hit "Inspect", ignored the "disconnected" message and read at console something about "Update on reload" option for service workers... I proceeded to `Application > Service Workers` and unchecked "Update on reload". So far it's working again, but I'm not 100% sure I've stumble on the right issue.
